bindings/gumjs/generate-runtime.py

```diff
@@ -155,10 +155,7 @@ def extract_source_map(filename, source_code):
     return (stripped_source_code, raw_source_map)
 
 def to_canonical_source_path(path):
-    if path.startswith('../node_modules'):
-        return 'frida/' + path[3:]
-    else:
-        return 'frida/' + path
+    return os.path.join('frida', path).replace("\\","/")
```

Comment:
Missing space after comma, and, this one is my fault: let's use double-quoted strings here – I've been mixing styles in this script. (The original style was to use double-quoted strings for strings, and single-quoted strings for pretend enum/symbol values.)